src/Serilog/Configuration/LoggerDestructuringConfiguration.cs

```diff
@@ -180,5 +184,23 @@ public LoggerConfiguration ToMaximumCollectionCount(int maximumCollectionCount)
             _setMaximumCollectionCount(maximumCollectionCount);
             return _loggerConfiguration;
         }
+
+        /// <summary>
+        /// When destructuring objects, allows to filter properties recorded in StructureValue.
+        /// </summary>
+        /// <param name="filter">Delegate used for filtering.</param>
+        /// <returns>Configuration object allowing method chaining.</returns>
+        public LoggerConfiguration AllowOnly(Func<PropertyInfo, object, bool> filter)
```

Comment:
@nblumhardt I could not find a really good name for these two extension methods.